src/Symfony/Component/Messenger/Transport/SqsExt/Connection.php

```diff
@@ -0,0 +1,362 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\SqsExt;
```

Comment:
should we make this a `amazon-sqs-messenger` bridge instead, to host the SQS-specific code (similar to what is done in mailer) ?

Reply:
True question... Same for lock stores, cache adapters, session handlers.... Maybe one days we'll provide a redis-pack that provide all bridges for redis...

Reply:
Yes! We should.

Reply:
Done